### DISCUSSION TITLE
ntfs-3g disabled?

### DISCUSSION THREAD

**Discussion by codethought** [Homebrew/discussions#1308] [Everyday usage] [answered]
Hello,

I was curious why ntfs-3g was disabled... and what a good alternative might be for use on Big Sur.

Thanks.

**Comment by carlocab** [comment, marked as answer]
It requires macfuse/osxfuse, which is no longer open source.

You can host a version of this formula in your own tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Reply by tlk** [reply]
Link to osxfuse licensing issue: https://github.com/osxfuse/osxfuse/issues/616#issuecomment-726427625

Somebody already made their own tap at https://github.com/darelover/homebrew-ntfs-3g

I wonder if https://github.com/Homebrew/homebrew-cask/blob/master/Casks/osxfuse.rb will be removed for the same reasons?

**Reply by codethought** [reply]
Thank you for the insights and information.  I've switched over to the tap at https://github.com/darelover/homebrew-ntfs-3g

**Reply by tlk** [reply]
Somebody already asked about the Homebrew Cask in https://github.com/osxfuse/osxfuse/issues/777#issuecomment-766150621 and the author answered:
> bfleischer: For the record, I do not consider Homebrew commercial software. The license change does not affect Homebrew.

**Reply by SMillerDev** [reply]
It affects homebrew because it is no longer under an open source license so we would be breaking our own rules if we used it.

**Reply by carlocab** [reply]
I think the issue was whether the license change also prevented distribution via Homebrew Cask. The author confirms that it does not.

**Reply by alebcay** [reply]
> I wonder if Homebrew/homebrew-cask@master/Casks/osxfuse.rb will be removed for the same reasons?

This Cask will remain because there is no requirement that Casks are open-source software. We ship plenty of proprietary software via Casks.

> bfleischer: For the record, I do not consider Homebrew commercial software. The license change does not affect Homebrew.

The issue is that the new license is not FSF/OSI/DFSG approved, which is the real litmus test for what we can include in Homebrew _Core_ (not Cask). This is due to Homebrew's relationship with the [SFC](https://sfconservancy.org/), and because we build bottles (binaries) for software in core, while Casks only re-distribute binaries built by someone else (usually the software vendors themselves), and Casks download directly from the software provider themselves (we don't do any sort of mirroring or file hosting for software provided via Casks).
